extensions/amp-analytics/0.1/test/test-instrumentation.js

```diff
@@ -87,4 +87,22 @@ describe('instrumentation', function() {
     expect(fnIdY.callCount).to.equal(1);
   });
 
+  it('should listen on custom events', () => {
```

Comment:
Can you add a test that shows how to listen to this from an analytics config. Also add docs?